### ISSUE TITLE
[Feature Request] Navigating with the arrow keys (up/down)

### ISSUE THREAD

**Issue by yet-another-average-joe** [Ghostkeeper/SettingsGuide#8]
Keyboard navigation with the arrow keys is missing. But the spacebar opens/closes subitems as expected. Arrow keys (up/down) would make the navigation much easier. No need to look at the list, point and click, and then go back to the contents. Could be read as a pdf.

Is this the standard behaviour ? It works the same in Cura, the arrow keys events being sent to the 3D viewer even when the focus is somewhere in the SideBar.

**Comment by Ghostkeeper** [comment]
This is implemented for version 2.1 of the plug-in. After clicking on any article in the sidebar, the focus will be on the list of articles. From then on you can use the arrow keys to navigate through the list.
